frontend/src/Settings/General/HostSettings.js

```diff
@@ -71,6 +72,22 @@ function HostSettings(props) {
         />
       </FormGroup>
 
+      <FormGroup
+        advancedSettings={advancedSettings}
+        isAdvanced={true}
+      >
+        <FormLabel>Instance Name</FormLabel>
+
+        <FormInputGroup
+          type={inputTypes.TEXT}
+          name="instanceName"
+          helpText="Instance name in tab and for Syslog app name"
+          helpTextWarning="Requires restart to take effect"
```

Comment:
Make sure this prompts the user to restart, currently it doesn’t, but reload applies it immediately, which feels a bit weird.